.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,81 @@
+name: Python package
+
+on:
+  push:
+  release:
+    types:
+      - published
+
+jobs:
+  build_wheels_linux:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: 'true'
+    - name: Build wheels
+      uses: pypa/cibuildwheel@v2.12.3
+      env:
+        CIBW_BEFORE_ALL: "sh linux-build.sh"
+        CIBW_BEFORE_BUILD: echo {project}/binaries
+        CIBW_SKIP: "cp36-* pp*"
+        CIBW_ARCHS_LINUX: auto64
+        CIBW_REPAIR_WHEEL_COMMAND: "LD_LIBRARY_PATH=/project/binaries auditwheel repair -w {dest_dir} {wheel}"
+    - uses: actions/upload-artifact@v3
+      with:
+        path: ./wheelhouse/*.whl
+
+  build_wheels_windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        arch:
+          - AMD64
+          - x86
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: 'true'
+    - uses: actions/setup-python@v4
+      with:
+        python-version: '3.8' 
+    - uses: ilammy/msvc-dev-cmd@v1
```

Comment:
this one I will need to dig into a little bit, to see how legit it is...

Is it recommended by the CIBW folks?

Reply:
It looks legit to me, although no mention of it in the CIBW docs that I can see.

I don't know much about Windows; is this needed so that `get-nasm.py` script works?

Reply:
It’s necessary to setup the environment for nmake.